sites/default/config_kiAoJEvYbDug-MYtyMpQ9xyHgrasKqDohb1kTzQwjqz_n72h6RnfHNJCvitDLKILomWJNFsmYg/sync/field.field.nica_entity.project_history.field_result.yml

```diff
@@ -0,0 +1,25 @@
+uuid: af63801f-8387-4d03-bc00-af52f741e446
+langcode: es
+status: true
+dependencies:
+  config:
+    - field.storage.nica_entity.field_result
+    - nica_entity.content_type.project_history
+id: nica_entity.project_history.field_result
+field_name: field_result
+entity_type: nica_entity
+bundle: project_history
+label: 'Resultado del Proyecto'
+description: ''
+required: false
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:taxonomy_term'
+  handler_settings:
+    target_bundles: {  }
```

Comment:
No default bundles set. Please configure projects and events to use the new vocabulary performance_result.